docs/LEARNING.md

```diff
@@ -2,6 +2,7 @@
 
 * [The Rust Programming Language](http://doc.rust-lang.org/stable/book/) is a great resource for getting started with Rust as well as diving deeper into specific features of Rust.
 * [Rust by Example](http://rustbyexample.com) shows you examples of the most common things you will be writing in Rust.
+* [Into_rust()](http://intorust.com/) "Screencasts for learning Rust."
```

Comment:
I was about to say what's with the extra parentheses after rust, but it's part of the name at http://intorust.com/about/ so okay.

Are the quotes necessary?

(Then again, someone may ask why the quotes are there for Rustlings and #rust-beginners... maybe they should all go)